syntax = "proto3";
package recastx;

import "google/protobuf/empty.proto";

service Imageproc {
  rpc SetDownsampling (Downsampling) returns (google.protobuf.Empty) {}

  rpc SetProjectionFilter (ProjectionFilter) returns (google.protobuf.Empty) {}
}

message Downsampling {
  uint32 col = 1;
  uint32 row = 2;
}

message ProjectionFilter {
  string name = 1;
}
